src/main/java/seedu/address/model/ReadOnlyAddressBook.java

```diff
@@ -34,4 +36,50 @@
      */
     ObservableList<Issue> getIssueList();
 
+    //=========== Alias =============================================================
+    /**
+     * Returns the current alias mapping.
+     *
+     * @return The mapping.
+     */
+    AliasMapping getAliasMapping();
+
+    /**
+     * Returns an Alias based on name.
+     *
+     * @param aliasName Name of the alias.
+     * @return The alias with the specified name.
+     */
+    Alias getAlias(String aliasName);
+
+    /**
+     * Deletes a user-defined alias from the current mapping.
+     *
+     * @param aliasName The name of the alias to be deleted.
+     */
+    void deleteAlias(String aliasName);
```

Comment:
Perhaps don't include delete as part of the read-only address book.

Reply:
Good catch, updated!